CMakeLists.txt

```diff
@@ -1,168 +1,186 @@
-cmake_minimum_required (VERSION 2.8)
```

Comment:
Please rebase instead of merging, as we see changes from master here.

Reply:
> Please rebase instead of merging, as we see changes from master here

I rebased, but still the changes are not appearing well.

Reply:
Its the "TBB almost ok" that is incorrect: it includes commits from master.

Reply:
Maybe "dos2unix CMakeLists.txt" will revert crlfs ?

Reply:
Should be better now..